public/app/features/explore/ContentOutline/ContentOutline.tsx

```diff
@@ -0,0 +1,82 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { useToggle } from 'react-use';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { reportInteraction } from '@grafana/runtime';
+import { useStyles2, ToolbarButton } from '@grafana/ui';
+
+import { useContentOutlineContext } from './ContentOutlineContext';
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    wrapper: css({
+      label: 'wrapper',
+      position: 'relative',
+      display: 'flex',
+      justifyContent: 'center',
+      height: '100%',
+      backgroundColor: theme.colors.background.primary,
+      marginRight: theme.spacing(1),
+      zIndex: theme.zIndex.sidemenu,
+    }),
+    content: css({
+      label: 'content',
+      position: 'sticky',
+      top: 0,
+      height: '85vh',
+    }),
+    buttonStyles: css({
+      textAlign: 'left',
+      width: '100%',
+    }),
+  };
+};
+
+const ContentOutline = () => {
```

Comment:
can we avoid a default export (as per https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#exports) ?

Reply:
also we are trying to move to function declarations instead of function expressions for components (although this is not enforced)
https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#how-to-declare-functional-components